docs/plugins/publishing.md

```diff
@@ -39,4 +39,18 @@ $ git push origin -u new-release
 And then create a PR. Once the PR is approved and merged into master, the master
 build will publish new versions of all bumped packages.
 
+### Include new changes in existing release PR
+
+If you want to include some late minute changes to an existing release PR,
+follow these instructions:
+
+```sh
+$ git checkout master
+$ git pull
+$ git checkout new-release
```

Comment:
maybe say `<release-branch-name>` to clarify that it's a dynamic part of the command so to speak